FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_mutual_auth/DemoTasks/MutualAuthMQTTExample.c

```diff
@@ -314,31 +316,48 @@ void vStartSimpleMQTTDemo( void )
 }
 /*-----------------------------------------------------------*/
 
+/*
+ * @brief The Example shown below uses managed MQTT APIs to create MQTT messages and
+ * send them over the mutually authenticated network connection established with the
+ * AWS IoT MQTT broker. This example is single threaded and uses statically
+ * allocated memory. It uses QoS1 for sending to and receiving messages from the broker.
+ *
+ * This MQTT client subscribes to the topic as specified in mqttexampleTOPIC at the
+ * top of this file by sending a subscribe packet and then waiting for a subscribe
+ * acknowledgment (SUBACK).This client will then publish to the same topic it
+ * subscribed to, so it will expect all the messages it sends to the broker to be
+ * sent back to it from the broker.
+ */
 static void prvMQTTDemoTask( void * pvParameters )
 {
     uint32_t ulPublishCount = 0U;
     const uint32_t ulMaxPublishCount = 5UL;
     NetworkContext_t xNetworkContext = { 0 };
     NetworkCredentials_t xNetworkCredentials = { 0 };
-    MQTTContext_t xMQTTContext;
+    MQTTContext_t xMQTTContext = { 0 };
     MQTTStatus_t xMQTTStatus;
 
     /* Remove compiler warnings about unused parameters. */
     ( void ) pvParameters;
 
+    /* Store the entry time of the demo application. This entry time is used for
+     * calculating the elapsed time in application, by the time utility function
+     * used by MQTT library. */
```

Comment:
The comment is confusing here in the phrases of "calculating the elapse time in application" which could suggest that the application is calculating the elapsed time...
"by the time utility function used by MQTT library" - This can be confusing to the reader about which time utility library is used by MQTT if they don't have the understanding of the library API yet.


```suggestion
    /* Set the entry time of the demo application. This entry time will be used
     * to calculate relative time elapsed in the execution of the demo application,
     * by the timer utility function that is provided to the MQTT library.
     */
```